sycl/include/sycl/ext/oneapi/matrix/matrix-jit-use.hpp

```diff
@@ -0,0 +1,669 @@
+//==------------------ matrix-jit-use.hpp - SYCL matrix ----------------*- C++
+//-*---==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// ===--------------------------------------------------------------------=== //
+
+#pragma once
+
+#include <CL/__spirv/spirv_ops.hpp>
+#include <sycl/detail/defines_elementary.hpp>
+#include <sycl/feature_test.hpp>
+
+namespace sycl {
+__SYCL_INLINE_VER_NAMESPACE(_V1) {
+namespace ext {
+namespace oneapi {
+namespace experimental {
+namespace matrix {
+
+// packed_a and packed_b will be replaced by packed once the use implementation
+// is stable.
+enum class matrix_layout { row_major, col_major, packed_a, packed_b, unused };
```

Comment:
can we make the change in this PR: packed_a and packed_b into packed?

Reply:
but https://github.com/intel/llvm/blob/sycl/sycl/include/sycl/ext/oneapi/matrix/matrix-tensorcore.hpp still has packed_a and packed_b. are u sure we should only keep packed?

Reply:
After discussion: packed_a and packed_b will be replaced by packed.

Reply:
Okay.
Please assign the values to the enum explicitly.